fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -530,6 +530,35 @@ struct DDQueue : public IDDRelocationQueue {
 		Future<Void> cancel;
 	};
 
+	struct MovingAverage {
+		int64_t previous;
+		int64_t total;
+		double interval;
```

Comment:
The type here should match the type for the corresponding knob in ServerKnobs.

Reply:
Now they are both `double`